dags/common/parsing/xml_extractors.py

```diff
@@ -119,6 +119,23 @@ def extract(self, article: ET.Element):
         return self.default_value
 
 
+class ConstantExtractor(IExtractor):
+    def __init__(self, destination, constant, required) -> None:
```

Comment:
why sometimes you add type annotations and sometimes you do not?

Reply:
Because I am not sticking with one way of writing code. Have to change this trait. Thank you for pointing it out :)